### PR TITLE
#5275: CODEOWNERS: update to include files relevant for ttnn team

### PR DESCRIPTION
**Pull request by cfjchu**

### CODE REVIEW

**Comment by ttjchu** [comment]
@tt-rkim can you upgrade @xanderchin to developer status?

**Comment by ttjchu** [comment]
@arakhmati @eyonland am I missing anything?

File: CODEOWNERS [hunk 1]
```diff
@@ -94,6 +94,12 @@ tt_eager/tt_dnn/op_library/composite/ @muthutt
 tt_eager/tt_dnn/op_library/complex/ @muthutt
 tt_eager/tt_dnn/op_library/backward/ @muthutt
 
+# eagar - tensor and op infra
+tt_eager/tt_dnn/op_library/operation_history.*pp @arakhmati @eyonland @ttjchu @xanderchin
+tt_eager/tt_dnn/op_library/operation.*pp @arakhmati @eyonland @ttjchu @xanderchin
+tt_eager/tt_dnn/op_library/program_cache.*pp @arakhmati @eyonland @ttjchu @xanderchin
+tt_eagar/tensor/ @arakhmati @eyonland @ttjchu @xanderchin
```
Comment (tt-rkim) [inline CODEOWNERS:101]: EAGAR

**Review by tt-rkim — CHANGES_REQUESTED** [review]

**Review by arakhmati — APPROVED** [review]

File: CODEOWNERS (hunk 1 above)
Comment (ttjchu) [reply CODEOWNERS]: You got me! Thanks!

**Comment by arakhmati** [comment]
> @arakhmati @eyonland am I missing anything?

Yes, add you and Xander to the every line with my name

**Review by tt-rkim — APPROVED** [review]

**Pull request merged**